src/electron/main/main.ts

```diff
@@ -31,6 +31,10 @@ const nativeHighContrastModeListener = new NativeHighContrastModeListener(
 log.transports.file.level = 'info';
 autoUpdater.logger = log;
 
+if (platformInfo.isLinux()) {
```

Comment:
@dbjorge, I'd love to get your thoughts on how we might be able to make this more selective

Reply:
I spent 15 minutes researching and came to the same conclusion you did. It looks like most utilities for detecting "parallels" vs not boil down to using [DMI](https://en.wikipedia.org/wiki/Desktop_Management_Interface) to check for a Parallels-specific BIOS entry, but the tools available to do that query seem to vary per distro (we probably shouldn't assume the presence of systemd, in particular, but the most common alternatives require more privileges than we're going to have).

Given that there isn't a substantial performance difference, I agree with just disabling it on linux generally. We have a very, very low proportion of linux users already, so I feel it's hard to argue a strong value prop for doing anything complicated here.

It would be nice to leave a comment here explaining why we're disabling it on linux with a pointer to the issue, though!

Reply:
Comment added